src/lib/run-dev.js

```diff
@@ -552,11 +553,27 @@ function createActionParametersFromRequest ({ req, actionInputs = {} }) {
     ...req.query,
     ...action.inputs
   }
-  if (req.is('application/*')) {
-    Object.assign(params, req.body)
-  } else {
-    params.__ow_body = req.body
+
+  const isJson = req.is('application/json')
+  const isFormData = req.is('application/x-www-form-urlencoded')
+  const isRaw = isRawWebAction(contextItem)
+
+  if (params.__ow_method === 'post' && req.body !== null) {
+    if (isRaw) {
+      if (isFormData) {
+        params.__ow_body = new URLSearchParams(req.body).toString() // convert json back to query string
+      } else if (isJson) {
+        params.__ow_body = Buffer.from(JSON.stringify(req.body)).toString('base64')
```

Comment:
nit: oportunity to merge the two `else if / else` branch? as utils.bodyTransformToRaw also supports objects right?

Reply:
thanks! will update

Reply:
fixed in 8893668532b89625c367622850bf00450e2d3b62